src/goto-instrument/code_contracts.h

```diff
@@ -216,6 +214,11 @@ class code_contractst
     const irep_idt &mode);
 };
 
+#define FLAG_LOOP_CONTRACTS "use-loop-contracts"
+#define HELP_LOOP_CONTRACTS                                                    \
+  " --use-loop-contracts\n"                                                    \
+  "                              simplify loops using loop contracts\n"
```

Comment:
Is simplify the correct wording here? How about "apply loop invariants on annotated loops"?

Reply:
Sure, either works. I'll update as per your suggestion.

I used "simplify" because we replace a loop with a conditional block, which is "simpler" control-flow structure.

Reply:
Again, this is possibly pedantic but I can think of at least two ways of using a loop contract: A. converting to an assertion and checking it holds on all iterations, B. using as an over-approximation of what the loop does by removing the loop, havocing modified variables and then assuming it.  Often these are done together but they really don't have to be.  I would only regard B. as simplifying the loop and then ... summarising or (over-)approximating the loop feel more accurate.  For example, if it is a short-ish loop that modifies arrays, unrolling might actually be "simpler" than using a quantifier.

Reply:
Good catch! :)
I totally agree. All of the instrumentation code only does (B), so that's why I called it a "simplification," but you're right that (A) can be "simpler" for unrolled short-ish loops.

Going to name this `--apply-loop-contracts` instead.